test/e2e/knative_serving_test.go

```diff
@@ -29,7 +29,11 @@ const (
 
 func TestKnativeServing(t *testing.T) {
 	caCtx := test.SetupClusterAdmin(t)
-
+	route, err := setupMetricsRoute(caCtx, "serving")
+	if err != nil {
+		t.Fatal("Failed to setup operator metrics route", err)
+	}
+	metricsPath := "http://" + route.Spec.Host + route.Spec.Path
```

Comment:
Let's add a new subtest after `deploy knativeserving cr and wait for it to be ready` that does this + the check and name it `verify health metrics work correctly`

Reply:
ok.